marketplace-kit-archive.js

```diff
@@ -107,9 +110,15 @@ const templateData = (path) => {
 
 program
   .version(version)
-  .option('--dir <dir>', 'files to be added to build', 'marketplace_builder')
   .option('--without-assets', 'if present assets directory will be excluded')
   .option('--target <target>', 'path to archive', process.env.TARGET || './tmp/marketplace-release.zip')
   .parse(process.argv);
 
-makeArchive(program.target, program.dir, program.withoutAssets);
+if (!fs.existsSync(APP_DIR)) {
```

Comment:
I would do the other way around, if there is app, use it, if not, fallback. 
Currently if there is both app and legacy, it will use legacy.